Plugin.pm

```diff
@@ -282,12 +282,58 @@ sub trackInfoMenu {
 	my $extid = $track->extid;
 	$extid ||= $url if $url =~ /^tidal:/;
 
-	return _objInfoMenu($client,
-		$extid,
-		$track->artistName || $remoteMeta->{artist},
-		$track->album ? $track->albumname : $remoteMeta->{album},
-		$track->title || $remoteMeta->{title},
-	);
+	my $artist = $track->remote ? $remoteMeta->{artist} : $track->artistName;
+	my $album  = $track->remote ? $remoteMeta->{album} : ( $track->album ? $track->album->name : undef );
+	my $title  = $track->remote ? $remoteMeta->{title} : $track->title;
+
+	my $search = cstring($client, 'SEARCH');
+	my $items = [];
+	push @$items, {
+		name => "$search " . cstring($client, 'ARTISTS') . " '$artist'",
```

Comment:
You could use a string with placeholders like we do in Qobuz to simplify this:

https://github.com/LMS-Community/plugin-Qobuz/blob/fe391178e2f1ff3926a6c5d44ab46bca2ac1b699/strings.txt#L185-L190

Reply:
The help text itself is a bit ugly  "Search Artists 'Nick Cave'" etc.

Reply:
User single `ARTIST` instead?

Reply:
Wouldn't `ARTIST` result in "Search Artist 'Nick Cave'"? (Haven't tested...)

Reply:
Yes, done.

![image](https://github.com/michaelherger/lms-plugin-tidal/assets/8558/f6dcd96e-4c79-4237-a4d4-527ac882bad2)

Reply:
Maybe `Search for 'Autechre' in Artists`, `Nach 'Oversteps' suchen unter Alben` bzw.  `Unter Alben nach 'Oversteps' suchen`.